cmd/showcommit.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/qri-io/ioes"
+	"github.com/qri-io/qri/lib"
+	"github.com/spf13/cobra"
+)
+
+// NewShowCommitCommand creates a new `qri search` command that searches for datasets
```

Comment:
:) you copypasta the search command too, eh?

Reply:
Bleh, I thought I fixed up all of the old names, guess I missed this.